amethyst_network/src/server/config.rs

```diff
@@ -37,6 +42,7 @@ impl Default for ServerConfig {
             udp_socket_addr: "0.0.0.0:0".parse().unwrap(),
             max_throughput: 5000,
             create_net_connection_on_connect: true,
+            laminar_config: None,
```

Comment:
Should the default be a laminar default config `Config::default()` rather than `None`?

Reply:
I have a reason for that but this might be better.